hibernate-core/src/test/java/org/hibernate/jpa/test/transaction/CloseEntityManagerWithActiveTransactionTest.java

```diff
@@ -83,6 +86,23 @@ public void tearDown() throws Exception {
 		}
 	}
 
+	@Test(expected = IllegalStateException.class)
+	@TestForIssue( jiraKey = "HHH-12487")
+	public void getTransactionOnClosedJtaEntityManagerShouldReturnNUll() {
```

Comment:
We should change the name of this test... it should not return null.  So either `testJtaGetTransaction` or `testJtaGetTransactionThrowsError`

Reply:
Also, neither this method nor `#getTransactionOnAJtaEntityManagerShouldReturnNUll` actually deal with an active transaction - so not sure they belong in a test named `CloseEntityManagerWithActiveTransactionTest`